code/game/gamemodes/objective.dm

```diff
@@ -768,5 +768,5 @@ var/list/potential_theft_objectives = subtypesof(/datum/theft_objective) \
 //wizard
 
 /datum/objective/wizchaos
-	explanation_text = "Wreak havoc upon the station as much you can. Send those wandless Nanotrasen scum a message!"
+	explanation_text = "Wreak havoc upon the station as much you can. Send those wandless NanoTrasen scum a message!"

```

Comment:
Then we should probably revisit that, as it is at all not something directly "codebase related" and was a decision that mark and mel did entirely on their own.